app/src/ui/branches/branches-container.tsx

```diff
@@ -65,17 +66,34 @@ export class BranchesContainer extends React.Component<
     }
   }
 
-  private onFilterKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+  private closePullRequrstFoldoutOnEsc = (
+    event: React.KeyboardEvent<HTMLElement>
+  ) => {
+    if (event.key === 'Escape') {
```

Comment:
We should check that no handler has taken care of this before us by doing `if (!event.defaultPrevented && e.key ===`